AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -300,15 +300,18 @@ bloody-disgusting.com,haberolduk.com,askdifference.com,multics.eu,pipeflare.io,t
 !#endif
 !
 ! NOTE: 'mdpDeBlocker'
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=noopjs,domain=texture-packs.com|xdarom.com
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=noopjs,domain=texture-packs.com|xdarom.com|katoikos.world
 ! #%#//scriptlet("abort-on-property-read", "mdp_deblocker")
 texture-packs.com#%#//scriptlet("abort-on-property-read", "mdpDeBlockerDestroyer")
 ! #%#//scriptlet('prevent-setTimeout', 'mdp-deblocker')
 wpnull7.com,xdarom.com#%#//scriptlet('prevent-setTimeout', 'mdp-deblocker')
 ! #%#//scriptlet("abort-on-property-read", "mdpDeBlocker")
+! https://github.com/AdguardTeam/AdguardFilters/issues/176499
+katoikos.world#%#//scriptlet('prevent-addEventListener', 'DOMContentLoaded', 'adsBlocked')
+katoikos.world#%#//scriptlet('prevent-setTimeout', 'adsBlocked')
```

Comment:
As I can see, it can be solved by the one scriptlet.
Please check this rule instead of the mentioned ones. The rule works fine on my end.
```adblock
katoikos.world#%#//scriptlet('prevent-eval-if', 'adsBlocked')
```

Reply:
Yes, this works on my side too, thank you, changed